db/migrate/20170110102731_remove_application_learning_period_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveApplicationLearningPeriodFromUsers < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :users, :application_learning_period
```

Comment:
Could you add the `:text` type so the migration becomes reversible (at least on the schema-level) please?